cps/cp35

```diff
@@ -0,0 +1,65 @@
+---
+cp: 35
+title: EVM compatibility on the Centrifuge Chain and Centrifuge App
+authors: annamehr
+contributors:
+proposal-type: CP4 technical
+date-proposed: 2023-02-24
```

Comment:
The proposal was created 2023-03-07 on the Forum

Reply:
1.cp35 - should be CP39
2. the name of the file should be cp39.md
3. status: polling